src/3_PB/VerbaliEsterni/VerbaleEsterno_240313/content.typ

```diff
@@ -0,0 +1,48 @@
+#import "meta.typ": inizio_incontro, fine_incontro, luogo_incontro, company
+#import "functions.typ": glossary, team
+#let participants = csv("participants.csv")
+#let participants_company = csv("participants_company.csv")
+
+= Partecipanti
+/ Inizio incontro: #inizio_incontro
+/ Fine incontro: #fine_incontro
+/ Luogo incontro: #luogo_incontro
+
+== Partecipanti di #team
+#table(
+  columns: (3fr, 1fr),
+  [*Nome*], [*Durata presenza*],
+  ..participants.flatten()
+)
+== Partecipanti di #emph[#company]
+#for member in participants_company.flatten() [
+  - #member
+]
+
+
+Durante l'incontro il gruppo ha chiarito con la Proponente alcuni dubbi che vengono esposti in seguito e ha condiviso con essa lo stato di avanzamento del progetto.
+
+== Stato di avanzamento del prodotto
+L'incontro si è aperto con la presentazione dell'attuale componente di visualizzazione e con una breve discussione riguardo ai progressi effettuati dall'ultimo SAL. Successivamente si è discussa la pianificazione temporale dell'attività di codifica per i rimanenti sprint.
+La proponente si è detta soddisfatta del lavoro che il team sta svolgendo e ha approvato la pianificazione presentata.
+
+== Candidatura revisione PB
+Il team ha messo al corrente la Proponente della recente decisione intrapresa di non affrontare la revisione CA e dunque di volersi limitare alla revisione PB pianificata orientativamente la prima settimana di Aprile. 
+
+== Test di integrazione
+Il team ha domandato alla Proponente riguardo quali sarebbero state le modalità preferibili per testare l'integrazione dei componenti.
```

Comment:
```suggestion
Il team ha domandato alla Proponente quali potrebbero essere le modalità preferibili per testare l'integrazione dei componenti del prodotto software.
```
più che le modalità preferibili direi gli strumenti più adatti ma it's up to you